server/commands/QueueNextCommand.ts

```diff
@@ -0,0 +1,46 @@
+import CommandDefinition from '../domain/CommandDefinition';
+import IoConnection from '../clients/IoConnection';
+import Song from '../domain/Song';
+import SongRepository from '../repositories/SongRepository';
+import QueueEventMessage from '../domain/event-messages/QueueEventMessage';
+import FetchVideoTitle from '../helpers/FetchVideoTitle';
+import SongService from '../services/SongService';
+
+async function createSongFromId(youtubeId: string) : Promise<Song> {
```

Comment:
This looks like something that could be extracted for additional reusability (same thing is used inside queue command service).

There is helpers module that looks like a good place for this (it looks like a good place for anything really...)